packages/react-client/src/DeviceManager.ts

```diff
@@ -130,11 +130,12 @@ export class DeviceManager
     this.storageConfig?.saveLastDevice(info);
   }
 
-  // todo `audioDeviceId / videoDeviceId === true` means use last device
-  public async start(deviceId?: string | boolean) {
-    const shouldRestart = !!deviceId && deviceId !== this.deviceState.media?.deviceInfo?.deviceId;
+  public async start(deviceId?: string) {
```

Comment:
**suggestion:** 
`const useLastDevice = deviceId === undefined;`
In this case `(deviceId || useLastDevice)` is almost always true, unless we pass as an empty string as an argument. (what is rather unwanted i guess)
```
  public async start(deviceId?: string) {
    const shouldRestart = !this.deviceState.media?.deviceInfo?.deviceId;
    const newDevice = deviceId ?? this.getLastDevice()?.deviceId ?? true;
```